MetaMask/metamask-extension
# Stop using 4bytes for contract deployment

## Explanation
This PR fixes a bug where we would try to show called method names for contract deployments. Since contract deployments have the bytecode as the data we can't look up method signatures using the first 4 bytes as you can with a normal contract interaction.




## Screenshots/Screencaps



### Before



### After



## Manual Testing Steps



1. Go to the test dapp
2. Deploy a contract
3. See that the transaction title is "contract deployment"

## Pre-Merge Checklist
